docs/guides/qiskit-addons-utils.ipynb

```diff
@@ -0,0 +1,445 @@
+      "text/plain": [
+       "<Figure size 623.998x471.567 with 1 Axes>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit_addon_utils.slicing import slice_by_gate_types\n",
+    "\n",
+    "slices = slice_by_gate_types(qc)\n",
+    "\n",
+    "# Recombine slices in order to visualize the partitions together\n",
+    "combined_slices = combine_slices(slices, include_barriers=True)\n",
+    "combined_slices.draw(\"mpl\", scale=0.6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57c844b5-cbe0-4369-80a6-57a2fc8840d9",
+   "metadata": {},
+   "source": [
+    "If your workload is designed to exploit the physical qubit connectivity of the QPU it will be run on, you can create slices based on edge coloring. The code snippet below will assign a 3-coloring to circuit edges and slice the circuit with respect to the edge coloring. (Note: this only affects non-local gates and single qubit gates will be sliced by gate type)."
```

Comment:
```suggestion
    "If your workload is designed to exploit the physical qubit connectivity of the QPU it will be run on, you can create slices based on edge coloring. The code snippet below will assign a three-coloring to circuit edges and slice the circuit with respect to the edge coloring. (Note: this only affects non-local gates. Single-qubit gates will be sliced by gate type)."
```